mx.py

```diff
@@ -4772,9 +4777,8 @@ def expand(value, context):
             'developer',
             'url',
             'licenses',
-            'licences',
             'defaultLicense',
-            'defaultLicence',
```

Comment:
Initially we were using the UK spelling. In an mx version we switched to US spelling. There is `mx_compat` support for that so that we can still load older suites.
I don't think we should remove these keys.

If there is a problem such as mx silently ignoring the old version of the keys, then we need a different solution were we first read the `mxversion` and then add the keys with the current spelling to the list of supported keys using `mx_compat`. But that would anyway be work for an other pull request.